future-apps/payroll/contracts/Payroll.sol

```diff
@@ -685,17 +685,19 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     function _transferTokensAmount(uint256 _employeeId, PaymentType _type, uint256 _totalAmount) internal returns (bool somethingPaid) {
         if (_totalAmount == 0) return false;
         Employee storage employee = employees[_employeeId];
+        address employeeAddress = employee.accountAddress;
         string memory paymentReference = _paymentReferenceFor(_type);
+
         for (uint256 i = 0; i < allowedTokensArray.length; i++) {
             address token = allowedTokensArray[i];
-            if (employee.allocation[token] != uint256(0)) {
-                uint128 exchangeRate = _getExchangeRate(token);
```

Comment:
I think dropping this line was not intentionally actually.

Reply:
The CI failed due to a compilation error since `exchangeRate` it's not declared.

Reply:
😅 Yes... messed up the merge conflict. Fixed in 9a602f2a702113c36a099d09cfbf67eacc8f7084.